src/app/my-page/my-page.component.css

```diff
@@ -0,0 +1,21 @@
+.main {
+    color: white;
```

Comment:
najlepiej na tym samym poziomie jak assets stworzyć  folder styles 
![image](https://user-images.githubusercontent.com/32534930/214399483-ab26aeba-a07b-4412-875c-4768943ccb52.png)
np. do pliku colors zadeklarować wszytskie zmienne z kolorami które są często uzywane w projekcie.
Breakpoints to wielkosci ekranów:
![image](https://user-images.githubusercontent.com/32534930/214399681-f22fa7cd-e017-4e4f-a803-4cfc88c050c8.png)
Fonts:

Reply:
Muszę się tego douczyć